src/extension.ts

```diff
@@ -344,10 +346,10 @@ async function _activateExtension(
   const docManager = DocumentMetadataManager.getInstance();
   context.subscriptions.push(docManager);
 
-  const provider = FlinkSqlCodelensProvider.getInstance();
+  const flinkProvider = FlinkSqlCodelensProvider.getInstance();
   context.subscriptions.push(
-    vscode.languages.registerCodeLensProvider(FLINK_SQL_LANGUAGE_ID, provider),
-    provider,
+    vscode.languages.registerCodeLensProvider(FLINK_SQL_LANGUAGE_ID, flinkProvider),
+    flinkProvider,
   );
```

Comment:
Future: we'll need to actually register the `AvroCodelensProvider` since it currently does nothing